include/grpc++/server_builder.h

```diff
@@ -62,6 +62,22 @@ class ServerBuilder {
  public:
   ServerBuilder();
 
+  struct SyncServerSettings {
```

Comment:
If we take the first option, then in the ServerBuilder constructor, call SetSyncOptions(SyncServerSettings()) to initialize values.